sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/AbstractModuleConfigurationTest.java

```diff
@@ -67,7 +67,7 @@ public void giveWarningsWhenGettingProtobufPathAndNoPropertyAvailable() {
 
     AbstractModuleConfiguration config = createAbstractModuleConfiguration(configuration);
     assertThat(config.protobufReportPaths()).isEmpty();
-    assertThat(logTester.logs(LoggerLevel.WARN)).containsOnly("Property missing: 'sonar.cs.analyzer.projectOutPaths'. No protobuf files will be loaded for this project.");
+    assertThat(logTester.logs(LoggerLevel.WARN)).containsOnly("Property missing: 'sonar.cs.analyzer.projectOutPaths'. No protobuf files will be loaded for project '<NONE>'.");
```

Comment:
Can you change mocking of this test to use some project key to see that the value is really used? The `<NONE>` is tested by the test below.